packages/sources/coinlore/src/endpoint/global.ts

```diff
@@ -17,6 +17,10 @@ const customParams = {
 export const execute: ExecuteWithConfig<Config> = async (request, config) => {
   const validator = new Validator(request, customParams)
   if (validator.error) throw validator.error
+  const endpoint = validator.validated.data.endpoint || config.defaultEndpoint
+  if (endpoint.toLowerCase() === 'globalmarketcap') {
```

Comment:
Something is strange here.

`path` and `field` are carrying the same purpose - to the path to key into the object.

We need to consolidate these across all adapters by going with one or the other. Let's talk about it in standup tomorrow and do this in a new branch & PR.